Lib/test/test_uuid.py

```diff
@@ -657,6 +658,10 @@ def testIssue8621(self):
 
             self.assertNotEqual(parent_value, child_value)
 
+    def test_uuid_weakref(self):
```

Comment:
Can you please add a short comment mentioning bpo-35701. Example: "bpo-35701: check that a weak reference to a UUID object can be created".